spring-boot-modules/spring-boot-basic-customization-2/src/main/java/com/baeldung/OncePerRequestFilter/OncePerRequestFilterApp.java

```diff
@@ -0,0 +1,12 @@
+package com.baeldung.OncePerRequestFilter;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+
+@SpringBootApplication(scanBasePackages = "com.baeldung.OncePerRequestFilter")
```

Comment:
correct this while changing the package name

Reply:
done